### PR TITLE
[onert-micro] Fix grad of Sparse Cross Entropy Loss

### PR DESCRIPTION
**Pull request by chunseoklee**
- Fix wrong sparse CCE loss grad computation

ONE-DCO-1.0-Signed-off-by: Chunseok Lee <chunseok.lee@samsung.com>

### CODE REVIEW

**Comment by chunseoklee** [comment]
@ljwoo94 I didnt test https://github.com/Samsung/ONE/pull/13637 with this PR :)

**Review by ljwoo94 — APPROVED** [review]
LGTM.
This modification makes similar shape of logic compare to that of `CrossEntropy` backward propagation.

I tested with testcase in #13637 and gained same accuracy result.
